ansible/roles/nginx/files/s142.okserver.org/icinga

```diff
@@ -0,0 +1,23 @@
+server {
+    server_name icinga.okserver.org;
+    index /wiki/index.php;
```

Comment:
This is what's causing the problem. Replace these two lines with the following

    index index.php;
    root /usr/share/icingaweb2/public/;